.changes/unreleased/Fixes-20231013-130943.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+body: resolve nested dependencies for packages installed using tarball method
```

Recommendation for clarity (make it easier for people to see right away if this change is relevant to them):
```suggestion
For packages installed with tarball method, fetch metadata to resolve nested dependencies
```